site/content/en/docs/overview/_index.md

```diff
@@ -35,6 +35,25 @@ A core design principle for Kueue is to avoid duplicating mature functionality i
+| [FlavorFungibility](/docs/concepts/cluster_queue/#flavorfungibility)                                                             | +              | +      | +     | +         | +          | +     | +          | +      | +   | +          | +      |
+| [MultiKueue](/docs/concepts/multikueue/)                                                                                         | +              | +      |       | +         | +          | +     | +          |        |     |            |        |
+| [MultiKueueBatchJobWithManagedBy](/docs/concepts/multikueue/#multikueuebatchjobwithmanagedby-enabled)                            | +              |        |       |           |            |       |            |        |     |            |        |
+| [PartialAdmission](/docs/tasks/run/jobs/#partial-admission)                                                                      | +              |        |       |           |            |       |            |        |     |            |        |
+| [ProvisioningACC](/docs/concepts/admission_check/)                                                                               | +              | +      | +     | +         | +          | +     | +          | +      | +   | +          | +      |
+| [QueueVisibility](/docs/tasks/manage/monitor_pending_workloads/pending_workloads_in_status/#enabling-feature-queuevisibility)    | +              | +      | +     | +         | +          | +     | +          | +      | +   | +          | +      |
+| [VisibilityOnDemand](/docs/tasks/manage/monitor_pending_workloads/pending_workloads_in_status/#enabling-feature-queuevisibility) | +              | +      | +     | +         | +          | +     | +          | +      | +   | +          | +      |
+| [PrioritySortingWithinCohort](/docs/concepts/cluster_queue/#flavors-and-borrowing-semantics)                                     | +              | +      | +     | +         | +          | +     | +          | +      | +   | +          | +      |
+| [LendingLimit](/docs/concepts/cluster_queue/#lendinglimit)                                                                       | +              | +      | +     | +         | +          | +     | +          | +      | +   | +          | +      |
+| [MultiplePreemptions](/docs/concepts/preemption/#preemption-strategies)                                                          | +              | +      | +     | +         | +          | +     | +          | +      | +   | +          | +      |
+| [Dynamic Reclaim](/docs/concepts/workload/#dynamic-reclaim)                                                                      | +              | +      |       |           |            |       |            |        | +   |            |        |
```

Comment:
It looks like most of the features that don't apply to everything are on the first rows.
Or did you use a different approach to sorting?

Reply:
Yeah, I think sorting on this way is better. Thanks!